cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -118,6 +118,9 @@ spec:
             - name: tcp-model
               containerPort: 9003
               protocol: TCP
+             {{- with .Values.kubecostModel.extraPorts }}
```

Comment:
Does query service have a separate heading other than `kubecostModel`?

Reply:
Yes. moving to kubecostDeployment.queryService.extraPorts, same reasoning as the aggregator comment.